cluster-autoscaler/cloudprovider/cloudifyprovider/node_group.go

```diff
@@ -390,11 +388,12 @@ func (clng *NodeGroup) String() string {
 }
 
 // NodeToNodeGroup - create cloudify node group
-func NodeToNodeGroup(client *cloudify.Client, deploymentID, groupName string) *NodeGroup {
+func NodeToNodeGroup(client *cloudify.Client, deploymentID, groupName string, nodeDataType string) *NodeGroup {
```

Comment:
You can use:  

client *cloudify.Client, deploymentID, groupName, nodeDataType string in function definition.(Don't need to describe that each param is string, you can say that all paremeters in list are strings.)

Reply:
@0lvin I think this will be right if all the params are string but client param is not string so I do not think so we can apply this here

Reply:
Look to your last *groupName string, nodeDataType string*

Reply:
Done